python/ray/autoscaler/_private/prom_metrics.py

```diff
@@ -0,0 +1,63 @@
+from prometheus_client import (
+    CollectorRegistry,
+    Counter,
+    Gauge,
+)
+
+
+# The metrics in this class should be kept in sync with
+# python/ray/tests/test_metrics_agent.py
+class AutoscalerPrometheusMetrics:
+    def __init__(self, registry: CollectorRegistry = None):
+        self.registry: CollectorRegistry = registry or \
+            CollectorRegistry(auto_describe=True)
+        self.pending_nodes: Gauge = Gauge(
+            "pending_nodes",
+            "Number of nodes pending to be started.",
+            unit="nodes",
+            namespace="autoscaler",
+            registry=self.registry)
+        self.started_nodes: Counter = Counter(
+            "started_nodes",
+            "Number of nodes started.",
+            unit="nodes",
+            namespace="autoscaler",
+            registry=self.registry)
+        self.stopped_nodes: Counter = Counter(
+            "stopped_nodes",
+            "Number of nodes stopped.",
+            unit="nodes",
+            namespace="autoscaler",
+            registry=self.registry)
+        self.running_nodes: Gauge = Gauge(
+            "running_nodes",
+            "Number of nodes running.",
```

Comment:
```suggestion
            "Number of worker nodes running.",
```

Reply:
If you have time @ijrsvt it is work grep replacing running_nodes with running_workers.

Reply:
SGTM!